nix/status-go/default.nix

```diff
@@ -58,8 +58,10 @@ let
   goBuildFlags = "-v";
   # status-go params to be set at build time, important for About section and metrics
   goBuildParams = {
-    Version = removePrefix "v" version; # Geth forces a 'v' prefix
     GitCommit = rev;
+    Version = if builtins.match versionRegex version != null
```

Comment:
LGTM but does anyone ever need the actual value here? For example, to verify that an app is actually build with a proper status-go? Or it can be verified in some other way?

Reply:
You mean if a dev wants to check the `status-go` version in the app, and all they will see is `develop`, instead of a commit? A fair point, but sounds pretty niche. Lets see what core devs and testers think about it.

Reply:
Maybe the syntax would be clearer if we can add some parenthesis around the function call?

Reply:
Added.